doc/environment_variables.md

```diff
@@ -89,7 +90,7 @@ some configuration as environment variables that can be set.
   - pending_tags - traces still-in-progress tags on completion queues
   - polling - traces the selected polling engine
   - polling_api - traces the api calls to polling engine
-  - subchannel - traces the connectivity state or refcount changes of subchannel
+  - subchannel_refcount - traces refcount changes of subchannel
```

Comment:
The pre-existing refcount tracers below don't have description after them. So this new one doesn't need any either.